# Configuration file for the dialogue
bot:
  default_param:
    user_id: "Micol"
    bot_name: "QTRobot"
    bot_alias: "DemoBot"
    region_name: "eu-west-2"



google:
  default_param:
    project_id: "agent1-ewhnht"
    language: "it"
    session_id: "test_bot"
    credential_path: "$(env HOME)/.gcp/private-keys.json/private-keys.json" #path where private keys are mounted


chat_gpt:
  default_param:
    model: "gpt-3.5-turbo"